atlp-rwanda/destructors-ec-bn
# #184637162 Notifications

### What does this PR do?
 
- A user will be notified in real-time when there is an update on status of the product

### Description of Task to be completed?
 
Provided a product
If the product lifecycle changes (added, removed, expired, bought, order-status is changed, etc)
the notification should be sent to the the right user

### How should this be manually tested?

- Clone the repository
- Change the directory to destructors-ec-bn by running cd destructors-ec-bn in you terminal
- Checkout ft-notifications-184637162 branch by running git checkout `ft-notifications-184637162` in you terminal
- open .env.example to check how you can create you .env file
- Create your .env file
- Then install dependencies and devdependencies used by running npm install in you terminal
- To create the models used in this project run `npm migrate`
- Run the server by running `npm run dev` in your terminal
- open you browser and in URL field type `http://localhost:3000/api/v1/notifications` .
- when the page opens  login with user token, login first with a registered user so that u can get the token to use.
- in the top right corner there is a notification bell click on it to get all notification for logged in user.

### Any background context you want to provide?



### What are the relevant pivotal tracker/Trello stories?

#184637162

#### Screenshots (if appropriate)
![Screenshot (86)](https://user-images.githubusercontent.com/101397608/233044727-46c02958-3564-4956-9fa7-34fcf97096ac.png)
